src/router/routes.ts

```diff
@@ -56,6 +58,16 @@ export const routes = [
       requiresAuth: true
     }
   },
+  {
+    path: '/create-rules',
+    name: RouteNames.CREATE_RULES,
+    component: CreateRules,
+    meta: {
+      step: 3,
+      label: 'Create Rules',
```

Comment:
Check whether we use the label. I suspect we don't. This was probably meant to be step text and we never cleaned it up.

Reply:
I don't believe we do, will remove them 👍